src/main/kotlin/no/brreg/informasjonsforvaltning/organizationcatalogue/service/OrganizationCatalogueService.kt

```diff
@@ -9,6 +9,7 @@ import no.brreg.informasjonsforvaltning.organizationcatalogue.mapping.updateValu
 import no.brreg.informasjonsforvaltning.organizationcatalogue.mapping.updateWithEnhetsregisteretValues
 import no.brreg.informasjonsforvaltning.organizationcatalogue.repository.OrganizationCatalogueRepository
 import org.springframework.data.repository.findByIdOrNull
+import org.springframework.scheduling.annotation.Scheduled
```

Comment:
Eg synes det er ok i denne omgang å løyse schedulering for dette problemet med Spring. På sikt bør vi forsøke å tenke ein eksplisitt event-drevet måte å gjere dette på.

Reply:
Fristet ikke så mye å ta inn rabbit kun for dette, men vi kan se på det en gang senere :+1: